state/execution.go

```diff
@@ -334,7 +339,7 @@ func execBlockOnProxyApp(
 	}
 	proxyAppConn.SetResponseCallback(proxyCb)
 
-	commitInfo := getBeginBlockValidatorInfo(block, store, initialHeight)
+	commitInfo := buildLastCommitInfo(block, store, initialHeight)
 
 	byzVals := make([]abci.Evidence, 0)
```

Comment:
The original PR was removing lines 344-347. I don't really remember why, could you please check whether these should be removed?

Reply:
Yeah I saw that. I wasn't planning to add that commit until after. We also changed abci.Evidence to abci.Misbehavior